templates/examples.go

```diff
@@ -15,48 +16,73 @@ import (
 )
 
 var (
-	//go:embed templates
+	//go:embed quickstart
 	files embed.FS
 
-	exampleBasePath = "https://github.com/coder/coder/tree/main/examples/templates/"
+	exampleBasePath = "https://github.com/coder/coder/tree/main/examples"
 	examples        = make([]Example, 0)
 	parseExamples   sync.Once
 	archives        = singleflight.Group{}
 )
 
 type Example struct {
```

Comment:
hmm should we change this to quickstart?

Reply:
@ericpaulsen I was referring to `exampleBasePath`

Reply:
Additionally, I think this URL would be wrong. I imagine we should change it to the following?

```
https://github.com/coder/coder/tree/main/templates/quickstart
```

Reply:
@bpmct yes that's correct

Reply:
done